examples/hybrid-compute/Quantinuum_hseries_hybrid_compute.ipynb

```diff
@@ -0,0 +1,1208 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[CX q[0], a[0]; CX q[1], a[0]; Measure q[0] --> c[0]; Measure a[0] --> syn[0]; Reset a[0]; CX q[1], a[0]; CX q[2], a[0]; Measure q[1] --> c[1]; Measure a[0] --> syn[1]; Measure q[2] --> c[2]; Reset a[0]; ClassicalExpBox syn[0], syn[1], syn_old[0], syn_old[1], syn_new[0], syn_new[1]; WASM syn_new[0], syn_new[1], pfu[0], pfu[1], pfu[2], pfu_old[0], pfu_old[1], pfu_old[2], _w[0]; CopyBits syn[0], syn[1], syn_old[0], syn_old[1]; ClassicalExpBox c[0], c[1], syn[0]; ClassicalExpBox c[1], c[2], syn[1]; ]"
+      ]
+     },
+     "execution_count": 24,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "circuit.add_classicalexpbox_bit(creg[0] ^ creg[1], [syn[0]])\n",
+    "circuit.add_classicalexpbox_bit(creg[1] ^ creg[2], [syn[1]])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[CX q[0], a[0]; CX q[1], a[0]; Measure q[0] --> c[0]; Measure a[0] --> syn[0]; Reset a[0]; CX q[1], a[0]; CX q[2], a[0]; Measure q[1] --> c[1]; Measure a[0] --> syn[1]; Measure q[2] --> c[2]; Reset a[0]; ClassicalExpBox syn[0], syn[1], syn_old[0], syn_old[1], syn_new[0], syn_new[1]; WASM syn_new[0], syn_new[1], pfu[0], pfu[1], pfu[2], pfu_old[0], pfu_old[1], pfu_old[2], _w[0]; CopyBits syn[0], syn[1], syn_old[0], syn_old[1]; ClassicalExpBox c[0], c[1], syn[0]; ClassicalExpBox c[1], c[2], syn[1]; ClassicalExpBox syn[0], syn[1], syn_old[0], syn_old[1], syn_new[0], syn_new[1]; ]"
+      ]
+     },
+     "execution_count": 25,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "circuit.add_classicalexpbox_register(syn_old ^ syn, syn_new)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[CX q[0], a[0]; CX q[1], a[0]; Measure q[0] --> c[0]; Measure a[0] --> syn[0]; Reset a[0]; CX q[1], a[0]; CX q[2], a[0]; Measure q[1] --> c[1]; Measure a[0] --> syn[1]; Measure q[2] --> c[2]; Reset a[0]; ClassicalExpBox syn[0], syn[1], syn_old[0], syn_old[1], syn_new[0], syn_new[1]; WASM syn_new[0], syn_new[1], pfu[0], pfu[1], pfu[2], pfu_old[0], pfu_old[1], pfu_old[2], _w[0]; CopyBits syn[0], syn[1], syn_old[0], syn_old[1]; ClassicalExpBox c[0], c[1], syn[0]; ClassicalExpBox c[1], c[2], syn[1]; ClassicalExpBox syn[0], syn[1], syn_old[0], syn_old[1], syn_new[0], syn_new[1]; WASM syn_new[0], syn_new[1], pfu[0], pfu[1], pfu[2], pfu_old[0], pfu_old[1], pfu_old[2], _w[0]; ]"
+      ]
+     },
+     "execution_count": 26,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "circuit.add_wasm_to_reg(\"decode3\", wasm_file_handler, [syn_new, pfu], [pfu_old])"
```

Comment:
are you talking about the add_wasm function at some point?

It might also be interesting to show in an additional step at the end how the ordering of the wasm ops is preserved and how this can be influenced by the user.

Reply:
the add_wasm function has been added to the document.

I will chat to you tomorrow about the second point